internal/db/migrations/2024-08-28.0-tss_transactions.sql

```diff
@@ -0,0 +1,37 @@
+-- +migrate Up
+
+CREATE TABLE tss_transactions (
+    transaction_hash VARCHAR(70) PRIMARY KEY,
+    transaction_xdr VARCHAR(400),
```

Comment:
What is the 400 based on?
I just tried creating a simple transaction with 2 operations and it resulted in a 512 characters long string.
```
AAAAAgAAAAD54NA+ZF4ieMZuoyyAsDJSH8EEylTn08xEecwNABU97AAAAMgAAAMdFHk3/wAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAxzZGZzZGFmZGFzZmEAAAACAAAAAQAAAAD54NA+ZF4ieMZuoyyAsDJSH8EEylTn08xEecwNABU97AAAAAEAAAAA+eDQPmReInjGbqMsgLAyUh/BBMpU59PMRHnMDQAVPewAAAABQVNEQQAAAAD54NA+ZF4ieMZuoyyAsDJSH8EEylTn08xEecwNABU97AAAAiGT/OIAAAAAAQAAAAD54NA+ZF4ieMZuoyyAsDJSH8EEylTn08xEecwNABU97AAAAA0AAAABU0RTRAAAAAD54NA+ZF4ieMZuoyyAsDJSH8EEylTn08xEecwNABU97AAAAAAUQ/0AAAAAAPng0D5kXiJ4xm6jLICwMlIfwQTKVOfTzER5zA0AFT3sAAAAAAAAAAyy1lsAAAAAAAAAAAAAAAAA
```

I think we should stick to `TEXT` in these VARCHAR columns as there is no real need to limit them imo. See [Postgres Wiki](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_varchar.28n.29_by_default).

Reply:
Yeap, changed all varchar fields to TEXT